src/components/ProfilePage.js

```diff
@@ -1,20 +1,63 @@
-import React, {useState} from 'react';
+import React, {useContext, useEffect, useState} from 'react';
 import './ProfilePage.css';
 import { Container, Form, Button, Row, Col} from 'react-bootstrap';
-import picLogScreen from '../assets/picLogScreen.png'
+import Navbar from "./Navbar";
```

Comment:
Navbar can be imported only in App.js instead of importing in each individual file